packages/gatsby-remark-prismjs/README.md

```diff
@@ -380,14 +380,14 @@ By default, every line gets a prompt appended to the start, this behaviour can b
 to the language.
 
 ````
-```bash{outputLines: 2-10,12}
+```shell{outputLines: 2-10,12}
```

Comment:
As shown in line 376 this needs to be `bash`

Reply:
fixed in:

- #23256 `fix(gatsby-remark-prismjs): Update useCommandLine`

Reply:
updated line 376 in df7ad89